### PR TITLE
QA for Emails

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**

- [ ] Fixing Mobile App delete endpoint. (Form)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [Delete Endpoint](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-b4a6db4f-d7cb-450a-951a-2488e3b92c7d?ctx=documentation)

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/firebase/auth/functions/index.js [hunk 1]
```diff
@@ -26,18 +27,18 @@
...
+async function sendDeleteConfirmationEmail(email, userId, token) {
   const mailOptions = {
     from: {
       name: "AirQo Data Team",
       address: process.env.MAIL_USER,
     },
     to: email,
     subject: "Confirm Account Deletion - AirQo!",
-    html: emailTemplate.deleteConfirmationEmail(email, userId, creationTime),
+    html: emailTemplate.deleteConfirmationEmail(email, userId, token),
```
[the remark this answers is not in the document]
Comment (Baalmart) [reply src/firebase/auth/functions/index.js:41]: Hi @BenjaminSsempala , any quick thoughts on this?
Do we do any input validation for this `email` field from the user?

File: src/auth-service/utils/email.templates.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ module.exports = {
 `;
   },
 
-  v2_emailVerification: (firstName, user_id, token) => {
+  v2_emailVerification: (email,firstName, user_id, token) => {
```
Comment (Baalmart) [inline src/auth-service/utils/email.templates.js:69]: Hi @BenjaminSsempala , under which use case is this being used?
Which endpoint?

**Review by Baalmart — COMMENTED** [review]
thanks @BenjaminSsempala , thanks for this.
I have left a few comments for your review.

File: src/auth-service/utils/email.templates.js (hunk 2 above)
Comment (BenjaminSsempala) [reply src/auth-service/utils/email.templates.js:69]: > Hi @BenjaminSsempala , any quick thoughts on this?
> Do we do any input validation for this email field from the user?

Hey @Baalmart , Yes the input validation is done in the form, which also has it's own script
Here's a link to the form  and the appscript
[Form](https://docs.google.com/forms/d/16AQre4iALbWD6MSd8cTZdHw5hAOt6oPf8zXmNkRoeV0/edit#responses)
[Appscript](https://script.google.com/u/2/home/projects/11W4BGQ_zIvOtBpUGGPz9ZAheBBTYvR5UWpHNrQIe-RCX6zofG9wRP8lD)

**Comment by BenjaminSsempala** [comment]
> Hi @BenjaminSsempala , any quick thoughts on this?
> Do we do any input validation for this email field from the user?

Hey @Baalmart , Yes the input validation is done in the form, which also has it's own script
Here's a link to the form  and the appscript
[Form](https://docs.google.com/forms/d/16AQre4iALbWD6MSd8cTZdHw5hAOt6oPf8zXmNkRoeV0/edit#responses)
[Appscript](https://script.google.com/u/2/home/projects/11W4BGQ_zIvOtBpUGGPz9ZAheBBTYvR5UWpHNrQIe-RCX6zofG9wRP8lD)

**Comment by Baalmart** [comment]
Thanks @BenjaminSsempala

File: src/auth-service/utils/email.templates.js (hunk 2 above)
Comment (Baalmart) [reply src/auth-service/utils/email.templates.js:69]: thanks @BenjaminSsempala

**Review by Baalmart — APPROVED** [review]
thanks @BenjaminSsempala

**Pull request merged**